python/ray/dashboard/modules/node/node_head.py

```diff
@@ -729,17 +730,19 @@ async def dump(self, req) -> aiohttp.web.Response:
                 **{key: data},
             )
 
-    async def run(self, server):
-        await asyncio.gather(
+    async def run(self):
+        await super().run()
+        coros = [
+            dashboard_utils.NotifyQueue.watch_and_notify_forever(),
```

Comment:
This is no longer needed?

Reply:
Removed. I forgot to remove `NotifyQueue` https://github.com/ray-project/ray/pull/51920